dap-etl/extraction/src/test/scala/org/broadinstitute/monster/dap/hles/HLESExtractionPipelineBuilderIntegrationSpec.scala

```diff
@@ -33,7 +33,7 @@ class HLESExtractionPipelineBuilderIntegrationSpec extends PipelineBuilderSpec[A
 
   val studyIdsRequest = RedcapRequestGeneratorParams(
     testArgs.apiToken,
-    HLESurveyExtractionPipeline.arm,
+    List("baseline_arm_1"),
```

Comment:
🔧 If we keep the constant in the related file, this would become:
```suggestion
    List(HLESurveyExtractionPipeline.arm),
```